other/edge-server/client/.gitignore

```diff
@@ -23,3 +23,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+docker-compose.yml
```

Comment:
I don't think we need this in the .gitignore, because with the new `edgectl` stuff, this file will get written somewhere outside of this directory.